lunes_cms/cms/models/feedback.py

```diff
@@ -0,0 +1,99 @@
+from html import escape
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from django.utils.safestring import mark_safe
+from django.utils.text import Truncator
+
+
+class Feedback(models.Model):
+    """
+    Contains a title, a description and a reference to the matter of feedback
+    """
+
+    content_type = models.ForeignKey(
+        ContentType,
+        limit_choices_to=models.Q(
+            app_label="cms", model__in=["discipline", "trainingset", "document"]
+        ),
+        on_delete=models.CASCADE,
+        verbose_name=_("content type"),
+        help_text=_("The content type this feedback entry refers to."),
+    )
+    object_id = models.PositiveIntegerField(
+        verbose_name=_("object id"),
+        help_text=_("The id of the object this feedback entry refers to."),
+    )
+    content_object = GenericForeignKey("content_type", "object_id")
+    content_object.short_description = _("refers to")
+
+    comment = models.TextField(
+        verbose_name=_("comment"),
+    )
+
+    read_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="feedback",
+        verbose_name=_("marked as read by"),
+        help_text=_(
+            "The user who marked this feedback as read. If the feedback is unread, this field is empty."
+        ),
+    )
+
+    created_date = models.DateTimeField(
+        auto_now_add=True,
+        verbose_name=_("submitted on"),
+        help_text=_("The time and date when the feedback was submitted."),
+    )
+
+    def content_object_link(self):
+        """
+        Include a link to the edit form of the content object in list display
+
+        :return: Link to the edit form of the content object
+        :rtype: str
+        """
+        # Get link to the admin form if the content object
+        admin_edit_link = reverse(
+            f"admin:cms_{self.content_type.model}_change", args=[self.object_id]
+        )
+        # Escape label because string is marked as safe afterwards
+        escaped_label = escape(str(self.content_object))
+        # Mark string as safe to allow html link tag
+        return mark_safe(f"<a href={admin_edit_link}>{escaped_label}</a>")
+
+    content_object_link.short_description = _("refers to")
+
+    def content_object_type(self):
+        """
+        Better readable representation of the content type
+
+        :return: Verbose name of the content object's model
+        :rtype: str
+        """
+        return self.content_object._meta.verbose_name
+
+    content_object_type.short_description = _("content type")
+    content_object_type.admin_order_field = "content_type"
```

Comment:
```suggestion
```
I think I noticed a better way to do this - probably, we should just overwrite the `__str__()`-method of the content type model by creating a new file `lunes_cms/cms/models/content_type.py` with the following content:
```
from django.contrib.contenttypes.models import ContentType

# Better representation of content type
ContentType.add_to_class("__str__", lambda c: c.name)
```
and then 
```
from . import group, content_type
```
here:
https://github.com/digitalfabrik/lunes-cms/blob/eed6b36fe5ca7f38197c0021d42a4567b7b5b2f6/lunes_cms/cms/models/__init__.py#L7